skrifa/src/scale/glyf/mem.rs

```diff
@@ -190,8 +190,8 @@ mod tests {
         };
         // Required size adds 4 bytes slop to account for internal alignment
         // requirements. So subtract 5 to force a failure.
-        let not_enough = outline_info.required_buffer_size() - 5;
+        let not_enough = outline_info.required_buffer_size(false) - 5;
```

Comment:
Hints::Off would be so much clearer :)

Reply:
Agreed :) I'm planning changes to our `Hinting` type which could be used for this purpose. In the meantime, I'd prefer not to conjure a new type just to throw it away tomorrow. Will make sure to address this in a follow up.

Reply:
#708 so I don't forget